v1/recipe/save_recipe.py

```diff
@@ -187,6 +193,14 @@ def update(self, instance):
         self._save_subrecipe_data(instance)
         self._save_tags(instance)
         instance.save()
+        if instance.photo:
+            new_img = instance.photo.path
+        if old_img and old_img != new_img:
+            try:
+                if os.path.exists(old_img):
+                    os.remove(old_img)
```

Comment:
I am not quite sure what will happen with the cache etc. For `FieldFile` there is a delete method. But not for ImageKit-Fields. Probably needs more elaboration if `os.remove` is sufficient.